lib/src/tabs/Tabs.tsx

```diff
@@ -25,14 +25,9 @@ const DxcTabs = ({
     if (activeTabIndex == null) {
       setInnerActiveTabIndex(newValue);
     }
-    if (typeof onTabClick === "function") {
-      onTabClick(newValue);
-    }
+    onTabClick(newValue);
```

Comment:
When an event is marked as optional in _types.ts_, we should add optional method chaining when it is called: `onCloseClick?.()` or it will throw a **ReferenceError**. Be careful when removing `typeof === "function"`, this line is also controlling that is not undefined or null.